_rules/video-only-description-track-ac7dc6.md

```diff
@@ -48,8 +48,8 @@ A video only element with a track element that contains descriptions.
 
 ```html
 <video controls>
-	<source src="../test-assets/rabbit-video/silent.mp4" type="video/mp4" />
-	<source src="../test-assets/rabbit-video/silent.webm" type="video/webm" />
+	<source src="../test-assets/rabbit-video/silent.mp4" type="video/mp4">
+	<source src="../test-assets/rabbit-video/silent.webm" type="video/webm">
 	<track kind="descriptions" src="rabbit-video-descriptions.vtt" />
```

Comment:
`<track>` is also a void element, so it shouldn't have the end slash, similar to `<source>`